### PR TITLE
8267796: vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/TestDescription.java fails with NoClassDefFoundError

### PR DESCRIPTION
**Pull request by luchenlin**
This PR contains a backport of
https://github.com/openjdk/jdk/commit/12e943091071b02b5756b57527a5f86a3d881cb8
Patch is not clean, this file test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/hs201t002.cpp has changed by other commits, I ignored these commits and only change code for this issue.
Other than that the rest of the code is clean.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] [JDK-8267796](https://bugs.openjdk.org/browse/JDK-8267796) needs maintainer approval
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8267796](https://bugs.openjdk.org/browse/JDK-8267796): vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/TestDescription.java fails with NoClassDefFoundError (**Bug** - P3 - Approved)

### Reviewers
 * [Martin Doerr](https://openjdk.org/census#mdoerr) (@TheRealMDoerr - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev.git pull/2707/head:pull/2707` \
`$ git checkout pull/2707`

Update a local copy of the PR: \
`$ git checkout pull/2707` \
`$ git pull https://git.openjdk.org/jdk11u-dev.git pull/2707/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 2707`

View PR using the GUI difftool: \
`$ git pr show -t 2707`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/2707.diff">https://git.openjdk.org/jdk11u-dev/pull/2707.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk11u-dev/pull/2707#issuecomment-2106709246)

### CODE REVIEW

File: test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/TestDescription.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```
Comment (TheRealMDoerr) [inline test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/TestDescription.java:2]: Should be "Copyright (c) 2018, 2022" (see original change).

**Review by TheRealMDoerr — COMMENTED** [review]
LGTM. Please fix the Copyright header.

File: test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/TestDescription.java (hunk 1 above)
Comment (Eitanull) [reply test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/TestDescription.java]: fixed

**Comment by luchenlin** [comment]
> LGTM. Please fix the Copyright header.

@TheRealMDoerr  it's fixed, please check, thanks.

**Review by TheRealMDoerr — APPROVED** [review]
Thanks!

**Pull request closed without merging**